projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.html

```diff
@@ -301,7 +302,10 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
   let-insuredObjectItem="insuredObjectItem"
 >
   <div class="col notice">
-    {{ insuredObjectItem?.formattedValue || insuredObjectItem?.value }}
+    {{
+      insuredObjectItem?.formattedValue ||
+        (insuredObjectItem?.value ? insuredObjectItem?.value : '/')
```

Comment:
Check if we use `-` instead of `/` for empty values?

Reply:
We should perhaps implement the same visual for non-existing values as in comparison table - span with class `fs-icon icon-remove`.